plugins/woocommerce/tests/performance/config.js

```diff
@@ -4,17 +4,17 @@ export const base_host = __ENV.HOST || 'localhost:8086';
 
 export const admin_username = __ENV.A_USER || 'admin';
 export const admin_password = __ENV.A_PW || 'password';
-export const admin_acc_login = __ENV.A_ACC_LOGIN || false;
+export const admin_acc_login = __ENV.A_ACC_LOGIN === 'true';
 
 export const customer_username =
 	__ENV.C_USER || 'customer@woocommercecoree2etestsuite.com';
 export const customer_password = __ENV.C_PW || 'password';
 export const customer_user_id = __ENV.C_UID || '2';
 
-export const cot_status = __ENV.COT || false;
+export const hpos_enabled = __ENV.HPOS !== 'false';
```

Comment:
> Asking because we usually need to change this file in order to run the performance tests

Yup, we do. But also, aside from manually setting the `cot_status` and `admin_acc_login ` in the file itself, we also support setting their values through the environment variables `COT` and `A_ACC_LOGIN`. And when someone uses those environment variables, that's when things won't go well. 

For example, if we set the env var `COT` to `true`, k6 will accept that value as a string `"true"` instead of a boolean `true`. Therefore, when k6 executes tests that have this if condition...

<img width="406" alt="Zl6CWUTH6r" src="https://github.com/woocommerce/woocommerce/assets/4509348/6fb418e7-e3be-4462-9f10-05c677d5bd2b">

...then the condition will always be `false`, since string `"true"` is not equal to boolean `true`. Therefore, the k6 tests will run in non-HPOS mode, even though we had properly set the `COT` environment variable to `true`.

Same goes with the `A_ACC_LOGIN` environment variable.

Reply:
Cool, thanks for the clarification @rodelgc! Just wanted to confirm that this was an intended change and not a temporary change for running the perf tests against a certain config that we forgot to revert before pushing the commit 👍